### PR TITLE
Wallpaper strings

### PR DESCRIPTION
**Pull request by electricRGB**

### CODE REVIEW

File: Client/Frontend/Strings.swift [hunk 1]
```diff
@@ -282,6 +284,25 @@ extension String {
...
+                public static let SwitchDescription = MZLocalizedString("Settings.Home.Option.Wallpaper.SwitchDescription", value: "Cycle through and update image without leaving the homepage.", comment: "In the settings menu, on the Firefox wallpaper customization screen, this is the string describing the switch button's function, which allows a user to toggle wallpaper switching from the homepage logo on or off.", lastUpdated: .v98)
+                public static let WallpaperUpdatedToastLabel = MZLocalizedString("Settings.Home.Option.Wallpaper.UpdatedToast", value: "Wallpaper Updated!", comment: "In the settings menu, on the Firefox wallpaper customization screen, this is the title of toast that comes up once the user changes wallpaper, which lets them know that the wallpaper has been updated.", lastUpdated: .v98)
```
Comment (nishant2718) [inline Client/Frontend/Strings.swift:293]: "once", or "when"? But go with whatever you think is more clear for Localizers 😅

Comment (electricRGB) [reply Client/Frontend/Strings.swift]: updated

File: Client/Frontend/Strings.swift [hunk 2]
```diff
@@ -282,6 +284,25 @@ extension String {
...
+                public static let FxAmethystWallpaper = MZLocalizedString("Settings.Home.Option.Wallpaper.Accessibility.AmethystWallpaper", value: "Firefox wallpaper, amethyst pattern, button", comment: "In the settings menu, on the Firefox wallpaper customization screen, this is the accessibility string for the amethyst firefox wallpaper.", lastUpdated: .v98)
+                public static let FxSunriseWallpaper = MZLocalizedString("Settings.Home.Option.Wallpaper.Accessibility.SunriseWallpaper", value: "Firefox wallpaper, sunrise pattern, button", comment: "In the settings menu, on the Firefox wallpaper customization screen, this is the title accessibility string for the amethyst firefox wallpaper.", lastUpdated: .v98)
+                public static let FxCeruleanWallpaper = MZLocalizedString("Settings.Home.Option.Wallpaper.Accessibility.CeruleanWallpaper", value: "Firefox wallpaper, cerulean pattern, button", comment: "In the settings menu, on the Firefox wallpaper customization screen, this is the title accessibility string for the amethyst firefox wallpaper.", lastUpdated: .v98)
```
Comment (nishant2718) [inline Client/Frontend/Strings.swift:303]: I think you might want to update the comment to indicate the correct pattern for these two.

**Review by nishant2718 — APPROVED** [review]
Small nits, but good to go after those updates.

**Pull request merged**